package.json

```diff
@@ -7,6 +7,7 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
+    "test:coverage:watch": "jest --coverage --watch",
```

Comment:
IMHO this isn't necessary – this should be what `test:watch` does. Unless there is a scenario where you want `watch`, but don't want `coverage`?